CONTRIBUTING.md

```diff
@@ -62,34 +68,46 @@ Finally, for _development dependencies only_, you must add an entry to
 `ci/environment-dev.yaml` with the same version constraint as in
 `pyproject.toml`.
 
-## First Steps to fix an issue or bug
+## First Steps to contribute
+
+- Read the documentation
+- Fork this repo (see "Development environment" section above for more)
+- Install environment (see "Development environment" section above for more)
+- Run the unit tests successfully in `main` branch:
+    - `make test`
+
+### ...to fix an issue or bug
 
-- Read the documentation (working on adding more)
-- create the minimally reproducible issue
-- try to edit the relevant code and see if it fixes it
-- submit the fix to the problem as a pull request
-- include an explanation of what you did and why
+- Create a GitHub issue with a minimally reproducible example (TODO: Link to SSCCE?)
+- Create a branch to resolve your issue
+- Run the unit tests successfully in your branch
+- Create one or more new tests to demonstrate the bug and observe them fail
+- Update the relevant code to fix the issue
+- Successfully run your new unit tests
 
-## First steps to contribute new features
+### ...to contribute a new feature
 
-- Create an issue to discuss the feature's scope and its fit for this package
-- run pytest to ensure your local version of code passes all unit tests
-- try to edit the relevant code and implement your new feature in a backwards compatible manner
-- create new tests as you go, and run the test suite as you go
-- update the documentation as you go
+- Create an issue and discuss the feature's scope and its fit for this package with the team
+- Create a branch for your new feature in your fork
+- Run the unit tests successfully in your branch
+- Write the code to implement your new feature in a backwards compatible manner
+- Create at least one test that exercises your feature and run the test suite as you go
+    - Example: _TODO: Add a good example of a small PR with a feature and a test._
```

Comment:
Should this be a draft PR until you address this TODO?

Reply:
#530